package.json

```diff
@@ -8,9 +8,17 @@
     "type": "git",
     "url": "git://github.com/woocommerce/woocommerce-google-analytics-integration.git"
   },
+  "dependencies": {
+    "@wordpress/hooks": "^3.22.0",
+    "@wordpress/i18n": "3.15.0"
+  },
   "devDependencies": {
+    "@wordpress/eslint-plugin": "^8.0.0",
+    "@wordpress/scripts": "^24.6.0",
+    "eslint": "^7.17.0",
+    "eslint-import-resolver-webpack": "^0.13.1",
```

Comment:
The `eslint-import-resolver-webpack` package is not required after https://github.com/woocommerce/woocommerce-google-analytics-integration/commit/ad25ad98f5cea683e9f8eef24f5126a25360df1f.

Reply:
Fixed here https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/261/commits/3b5ea432cb3e9bc07ad2c8a19c39b69eefc07310